src/main/java/io/papermc/hangar/scanner/check/method/OpenConnectionMethodCheck.java

```diff
@@ -21,6 +21,8 @@ public MethodCheckResult check(MethodInsnNode insnNode, MethodNode methodNode, C
         } else if (insnNode.name.equals("send") && insnNode.owner.equals("java/net/http/HttpClient")) {
             // jdk http client
             return new MethodCheckResult(Severity.LOW, methodNode, classNode, "found open connection (jdk http) call");
+        } else if (insnNode.name.equals("<init>") && insnNode.owner.equals("java/net/InetSocketAddress")) {
+            return new MethodCheckResult(Severity.HIGH, methodNode, classNode, "defines a socket to connect to a server");
```

Comment:
this shouldn't be high, I would say medium, since its less common

Reply:
Be fare